docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -162,6 +166,11 @@ P=2 would only receive traffic if the combined health of P=0 + P=1 was less than
 | 25          |  25         |  100       | 25%            |   25%          |   50%          |
 +-------------+-------------+------------+----------------+----------------+----------------+
 
+To sum this up in pseudo algorithms:
+percent load to P=0 == min(100, health(P=0) * 100 / total_health)
+health(P=X) == (1.4 * healthy_PX_backends / total_PX_backends)
+total_health == min(100, Σ(health(P=0)...health(P=x));
+percent load to P=X == 100 - Σ(percent_load(P0)..percent_load(Px-1))
```

Comment:
@htuch this what you were looking for?

Suggestions for improvement welcome.

Reply:
Yes, this is great! Please verify that when the RST turns to HTML, this is all readable.